src/adder.rs

```diff
@@ -108,6 +124,25 @@ fn add_in_generic_value(generic_value: &mut Value, last_field: &str, toml: Item)
     }
 }
 
+fn handle_add_with_table_header_path(
+    table_header_path: &str,
+    dotted_path: &str,
+    value: &str,
+    doc: &mut DocumentMut) -> Result<()> {
+    let table_header_path_vec = table_header_path
+        .split('/')
+        .map(|s| s.to_string())
+        .collect::<Vec<String>>();
+    let dotted_path_vec = dotted_path
+        .split('/')
+        .map(|s| s.to_string())
+        .collect::<Vec<String>>();
```

Comment:
again we don't need strings here, should be fine to collect into `Vec<&str>`

Reply:
I tried that, but that's going to trigger a bunch of changes. I'd prefer to do it in a separate cleanup pr.